CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## securesystemslib v0.10.9
+
+* Add `debian` directory (and files) that can be used to package a .deb file.
+
+* Modify the functions that import keys so that the key file's path is
+  specified in the function's prompt.
```

Comment:
Nitpick: How about something like:
*"Modify functions that generate or import keys so that the key file's path is shown if the function prompts for a password."*